src/vs/workbench/parts/files/electron-browser/fileActions.ts

```diff
@@ -724,6 +722,43 @@ class BaseDeleteFileAction extends BaseFileAction {
 			});
 		});
 	}
+
+	private getDeleteMessage(distinctElements: Array<ExplorerItem>, deletePermanentaly: boolean = false): string {
+		const key = deletePermanentaly ? 'DeleteMessage' : 'MoveTrash';
+		const messages = this.getMessages(deletePermanentaly);
+		let message: string = '';
+
+		if (distinctElements.length > 1) {
+			if (distinctElements[0].isDirectory) {
+				message = getConfirmMessage(nls.localize(`confirm${key}MessageMultipleDirectories`, messages[`confirm${key}MessageMultipleDirectories`], distinctElements.length), distinctElements.map(e => e.resource));
```

Comment:
@bpasero I am not sure about interpolating a variable here. But I couldn't find any other easier way to remove the duplicate code, if used the keys as it is..